engines/tinsel/detection.cpp

```diff
@@ -110,7 +111,10 @@ bool TinselMetaEngine::hasFeature(MetaEngineFeature f) const {
 		(f == kSupportsListSaves) ||
 		(f == kSupportsLoadingDuringStartup) ||
 		(f == kSupportsDeleteSave) ||
-		(f == kSimpleSavesNames);
+		(f == kSimpleSavesNames) ||
+		(f == kSavesSupportMetaInfo) ||
+		(f == kSavesSupportThumbnail) ||
```

Comment:
Was kSavesSupportThumbnail incorrectly added, or is that something you still want to experiment with?

Reply:
If I remove that flag then the SaveDate info does not get shown.

Reply:
Huh. You're right. I'll merge this in then.